src/getting-started/starting-stack.md

```diff
@@ -84,16 +84,8 @@ replica_count = 2
 
 By following these steps, you ensure that your CLI can be run from any location and still correctly locate and load its configuration files on Windows, macOS, and Linux.
 
-## Choose a track
-There are 2 tracks that developers can take when using `arch-cli`:
-1. [Program-only]
-  - A single-node execution environment for testing program logic.
-2. [Complete dApp]
-  - A complete, multi-node execution environment with consensus for testing a full-stack web dApp.
+## Start the validator
```

Comment:
Did you add the command for starting the validator?

Reply:
Yeah it's [here](https://github.com/Arch-Network/book/pull/13/files#diff-ae03163be26527076a8c0e976466e0e09e0822d3bbb4791c983989ca58335cfeR98)

Reply:
Oh I see. Great.